lib/order.rb

```diff
@@ -0,0 +1,47 @@
+require_relative 'customer'
+class Order 
+  attr_reader :id 
+  attr_accessor :products, :customer, :fulfillment_status
+  FULFILLMENT_VALID_STATUS = [:paid, :processing, :shipped, :complete, :pending].freeze
+  def initialize(id, products, customer, fulfillment_status = :pending)
+    @id = id 
+    @products = products
+    @customer = customer
+    @fulfillment_status = fulfillment_status
+    raise ArgumentError if !FULFILLMENT_VALID_STATUS.include?(fulfillment_status)
```

Comment:
This is a little easier to read if you use an `unless`.  Also, it's helpful to include a message with your errors that includes the bad argument.

```suggestion
    raise ArgumentError, "Bad status: #{status}" unless FULFILLMENT_VALID_STATUS.include?(fulfillment_status)
```